cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -452,6 +453,10 @@ public static int _main(String[] _args) throws Exception {
 
         String user = null;
         String auth = null;
+
+        String userIdEnv = System.getenv("JENKINS_USER_ID");
+        String tokenEnv = System.getenv("JENKINS_API_TOKEN");
```

Comment:
I am not satisfied with the name as it can contain API Token OR password. I don't have a better name at hand but I feel it so confusing.

WDYT to use a single env variable like `JENKINS_CLI_AUTH` and just checking the `<userId>:<tokenOrPassword>` format instead of the double check on their existence ? That could simplify a bit the code and reduce the very hard task to find a better name for the second variable ;)